Ginger/GingerCoreCommon/VariablesLib/VariableSelectionList.cs

```diff
@@ -122,13 +148,43 @@ public override void ResetValue()
             if (OptionalValuesList.Count > 0)
             {
                 Value = OptionalValuesList[0].Value;
+                CurrentValueIndex = 0;
             }
 
         }
 
         public override void GenerateAutoValue()
         {
-            //NA
+            string trimString = "Options: ";
```

Comment:
i didn't remmber we talked on allow random value, i don't really see the value of it- need to review the user story again.
I think this method just needs to find the location of current selected value and return the next value in list.

Reply:
Need to clean also all work on string with ','